openpower/package/libflash/libflash.mk

```diff
@@ -5,8 +5,10 @@
 ################################################################################
 
 LIBFLASH_VERSION = v5.7-76-g830fc9a0ed0a
-
 LIBFLASH_SITE = $(call github,open-power,skiboot,$(LIBFLASH_VERSION))
+HOST_LIBFLASH_VERSION = 830fc9a0ed0a4066129cfafd1447dc40a66b8700
```

Comment:
LIBFLASH_VERSION was at version that didn't have the pflash options I needed. Looks like it was bumped since I started. I'll combine this into one version.